gpii/node_modules/settingsHandlers/src/WebSocketsSettingsHandler.js

```diff
@@ -0,0 +1,154 @@
+/*!
+  GPII Web Sockets Settings Handler
+
+  Copyright 2014 Emergya
+
+  Licensed under the New BSD license. You may not use this file except in
+  compliance with this License.
+
+  You may obtain a copy of the License at
+  https://github.com/gpii/universal/LICENSE.txt
+*/
+
+/*global require*/
+
+(function () {
+
+    "use strict";
+
+    var fluid = require("infusion"),
+        gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.settingsHandlers");
+    fluid.registerNamespace("gpii.settingsHandlers.WebSocketsSettingsHandler");
+
+    // This global object keeps in memory the connected clients.
+    // Clients are added and removed respectively through addBrowser and
+    // removeBrowser functions
+    //
+    gpii.settingsHandlers.WebSocketsSettingsHandler.clients = {};
```

Comment:
Still missing some things (such as the big TODO), but I've implemented these changes (https://github.com/javihernandez/universal/commit/450ff32744a8f2a5196f43d4d23a3f2a4daf8faa).
@amb26, better?